src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerAddNote.java

```diff
@@ -101,13 +102,16 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
 
             caseData.setNotes(listValues);
         } else {
+            AtomicInteger listValueIndex = new AtomicInteger(0);
```

Comment:
Does this need to be an AtomicInteger?

Reply:
doesn't need to be an atomic integer we can do it in traditional counter way but thought will use this as it will auto increment :)